woocommerce-services.php

```diff
@@ -883,18 +845,16 @@ public function load_admin_dependencies() {
 			add_action( 'admin_notices', array( WC_Connect_Error_Notice::instance(), 'render_notice' ) );
 			add_action( 'admin_notices', array( $this, 'render_schema_notices' ) );
 
-			if ( ! $this->is_wc_shipping_activated() ) {
```

Comment:
We are loading `http://localhost/wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings` regardless so this condition is removed right? So that user can edit the live rate packages directly.

Reply:
Yep. Removed so the WCS&T settings page is accessible again, however I did also remove the link to the old settings page within the class controlling that settings page: https://github.com/Automattic/woocommerce-services/pull/2812/files#diff-45ab2d6757efe6316bd243c6eb0cf9194e30ddd4c01b98de3ed0e4a9b0aec1a9